pkg/loader/compose/compose.go

```diff
@@ -374,7 +374,10 @@ func (c *Compose) LoadFile(files []string) (kobject.KomposeObject, error) {
 		serviceConfig.Tty = composeServiceConfig.Tty
 		serviceConfig.MemLimit = composeServiceConfig.MemLimit
 		serviceConfig.TmpFs = composeServiceConfig.Tmpfs
-		komposeObject.ServiceConfigs[name] = serviceConfig
+		komposeObject.ServiceConfigs[normalizeServiceNames(name)] = serviceConfig
+		if !reflect.DeepEqual(komposeObject.ServiceConfigs[normalizeServiceNames(name)], komposeObject.ServiceConfigs[name]) {
```

Comment:
That reflect seems little bit too much :-(
Can same think be done a bit easier without reflect and more readable?  
Why compere whole struct? Comparing just names should be enough no?